Add quirk to disable ISOC transfers for device
"ID 1131:1001 Integrated System Solution Corp. KY-BT100 Bluetooth Adapter"
http://qa.mandriva.com/show_bug.cgi?id=30638

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
---
 drivers/bluetooth/btusb.c |    3 +++
 1 file changed, 3 insertions(+)

(status: submitted upstream to bluez-devel ML on 2007-09-29)

--- linux-2.6.28/drivers/bluetooth/btusb.c.orig	2009-02-11 10:29:20.000000000 -0500
+++ linux-2.6.28/drivers/bluetooth/btusb.c	2009-02-11 10:30:27.000000000 -0500
@@ -120,6 +120,9 @@ static struct usb_device_id blacklist_ta
 	/* Kensington Bluetooth USB adapter */
 	{ USB_DEVICE(0x047d, 0x105e), .driver_info = BTUSB_WRONG_SCO_MTU },
 
+	/* ISSC Bluetooth Adapter v3.1 */
+	{ USB_DEVICE(0x1131, 0x1001), .driver_info = BTUSB_BROKEN_ISOC },
+
 	/* RTX Telecom based adapters with buggy SCO support */
 	{ USB_DEVICE(0x0400, 0x0807), .driver_info = BTUSB_BROKEN_ISOC },
 	{ USB_DEVICE(0x0400, 0x080a), .driver_info = BTUSB_BROKEN_ISOC },
